### ISSUE TITLE
TypeError: Module specifier, 'lit' does not start with "/", "./", or "../".

### ISSUE THREAD

**Issue by fleetcoder** [lit/lit#4181]
### Which package(s) are affected?

Lit Core (lit / lit-html / lit-element / reactive-element)

### Description

Hi Lit experts, thank you for checking out this issue, it is only Safari that causes a problem, I am wondering if there is a workaround? the code below causes the   **TypeError: Module specifier, 'lit' does not start with "/", "./", or "../".**   error in Safari:

```html
<!DOCTYPE html>
<head>
<script type="importmap">
  {
    "imports": {
      "lit": "https://cdn.jsdelivr.net/gh/lit/dist@2/core/lit-core.min.js"
    }
  }
</script>
<script type="module">
import {html, css, LitElement} from 'lit';

export class SimpleGreeting extends LitElement {
  static styles = css`p { color: blue }`;

  static properties = {
    name: {type: String},
  };

  constructor() {
    super();
    this.name = 'Somebody';
  }

  render() {
    return html`<p>Hello, ${this.name}!</p>`;
  }
}
customElements.define('simple-greeting', SimpleGreeting);
</script>
</head>
<body>
  <h1>Import maps: Using a CDN that does not transform</h1>
  <simple-greeting name="World"></simple-greeting>
</body>
```

### Reproduction

https://lit.dev/playground/#gist=0c4f66396b333c82cb27e7dd3b2129c0

### Workaround

have not found a workaround

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

2.0

### Browser/OS/Node environment

Browser: Safari 16.3
OS: 13.2.1

**Comment by justinfagnani** [comment]
This works in Safari 16.6 for me with no errors.

**Comment by justinfagnani** [comment]
Looks like import maps landed in Safari 16.4: https://caniuse.com/import-maps

**Comment by fleetcoder** [comment]
thank you @justinfagnani - this is very helpful, i really appreciate the reply! cheers. i (eventually) want to make a build that will work for older browsers too and also allow creating new Lit components without a build step - but this will work for now thanks again.
